fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -455,7 +455,7 @@ public Integer readResourceTypeId(String resourceType) throws FHIRPersistenceDBC
             resultSet = stmt.executeQuery();
             dbCallDuration = (System.nanoTime()-dbCallStartTime)/1e6;
             if (log.isLoggable(Level.FINE)) {
-                log.fine("DB search by ids complete. executionTime=" + dbCallDuration + "ms");
+                log.fine("DB search by ids complete. SQL=" + idQuery.toString() + "  executionTime=" + dbCallDuration + "ms");
```

Comment:
Should a `[` and `]` be used to delineate the idQuery output?

Reply:
in this case we didn't have a SqlQueryData object, so yeah, wrapping the query in square brackets could help to delineate it.